devicetypes/smartthings/ecolink-wireless-siren.src/ecolink-wireless-siren.groovy

```diff
@@ -68,13 +68,17 @@ def parse(String description) {
 		result = zwaveEvent(cmd)
 		
 		if(result!=null) {
-			createEvent(result)
+			//device sends 'OK' message when is plugged in
+			if(!result.contains('OK')) {
```

Comment:
where on earth does this "OK" come from?

Reply:
Yesterday I was in deep concern about this 'OK'. But today I cannot get it anymore.Let's assume that I made huge mistake.